### PR TITLE
update readme and remove precompiled targets

### PR DESCRIPTION
**Pull request by v-almonacid**
I also remove the precompiled targets since they are outdated and only include v1 bindings (Byron era).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,13 +85,14 @@ Make sure the rust targets for the platform you will work on (android/iOS) have
 
 1. `yarn install`
 1. `yarn setup_configs` - links libraries to ios testnet build configurations
+1. When building on iOS: `cd ios && pod install`
```
Comment (nicarq) [inline README.md:88]: I would keep this in the iOS. we still mention about installing dependences which is what we do with `pod install`

Comment (ashisherc) [reply README.md:88]: `pod install` requires `yarn install` to be done first. so we should add a setup step, which does common things at the first

Comment (nicarq) [reply README.md:88]: ah great point!

**Review by nicarq — APPROVED** [review]

**Review by nicarq — CHANGES_REQUESTED** [review]
.

**Review by nicarq — APPROVED** [review]

**Pull request merged**